### ISSUE TITLE
Changing color theme

### ISSUE THREAD

**Issue by bergur** [pratikborsadiya/vali-admin#50]
How can I change the color theme from primary color to any of the other colors on the color palette (e.g. purple or green ) like is shown in some of the screenshots?

**Comment by pratikborsadiya** [comment]
Hi @bergur ,

Follow the official [documentation](https://pratikborsadiya.in/blog/vali-admin/) to know how to customize the theme.

**Comment by bergur** [comment]
Ok sorry let me rephrase. Is there no other way of changing the primary color than to edit the value of the $primary-color in_vars.scss and build? (or find-replace in css).

Is there is no class that I can put in body tag or something to switch to another color-theme?

**Comment by pratikborsadiya** [comment]
There is no other way to changes primary colour other then the two ways which you have mentioned above. I am curious though about why you need another way when there are two ways available through which you can accomplish what you want.

**Comment by bergur** [comment]
The reason is that I want different color-themes for different pages: 

For example
 Home - Primary
 Tech - Green
 News - Purple

etc
